db/fixtures/conditions.yml

```diff
@@ -0,0 +1,24 @@
+---
+- :name: no_high_severity_failures
+  :description: Does not have cve failures with high severity
```

Comment:
Actually, this condition description is very vague.  It implies we can check for every CVE known even into the future, and that's not accurate.  Since it's really only the CVEs known by OpenSCAP, perhaps "OpenSCAP" should be in the name and description.